### PR TITLE
Added Chat to the Example Sync Server and Client

### PR DESCRIPTION
**Pull request by jgoldin-skillz**
- Added Chat example to server/client sync example
Players can now send example emoji Chats to each other during gameplay
- Updated SSL documentation in readme
- Updated TICK documentation in readme

### CODE REVIEW

File: Sync Server SDK Client Example/Assets/Scripts/Sync/Gameplay/SyncGameController.cs [hunk 1]
```diff
@@ -157,6 +157,11 @@ private void Update()
                     on(OpponentResumed.GetRootAsOpponentResumed(byteBuffer));
                     break;
 
+                case Opcode.Chat:
+                    Debug.Log("Chat receigedf");
```
Comment (TGier) [inline Sync Server SDK Client Example/Assets/Scripts/Sync/Gameplay/SyncGameController.cs]: "received"?

File: Sync Server SDK Client Example/Assets/Scripts/Sync/UI/ChatObject.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    public void Start()
+    {
+        gameObject.SetActive(false);
+    }
+
+    public void Show(Sprite chatSprite)
+    {
+        // TODO: visual polish with animations
```
Comment (TGier) [inline Sync Server SDK Client Example/Assets/Scripts/Sync/UI/ChatObject.cs]: Do we want to leave this in?

**Review by TGier — APPROVED** [review]
👍

**Review by chedsadakon — COMMENTED** [review]
Chedsadakon

**Pull request merged**